playbooks/roles/mongo_3_2/defaults/main.yml

```diff
@@ -95,6 +95,7 @@ MONGO_BACKUP_NODE: "" # note: most likely the ip address of the instance on whic
 MONGO_BACKUP_AUTH_DATABASE: ""
 MONGO_BACKUP_SNAPSHOT_DESC: "mongo-backup"
 MONGO_BACKUP_PRUNE_OLDER_THAN_DATE: ""  # passed to `date -d`; should be a relative date like "-30days"
+MONGO_BACKUP_SNITCH_URL="" # optional url to send a GET request to upon successful completion of a backup
```

Comment:
@swalladge Let's change this to explicitly mention DMS :) And I think it's OK to omit technical details about the type of request that is being sent:

```
MONGO_BACKUP_SNITCH_URL="" # Optional URL that will be used to ping Dead Man's Snitch upon successful completion of a backup.
```

Reply:
Ok if you think that will be clearer. I specifically worded it as I did because it doesn't need to be tied to the DMS service; it can use any service that works by sending a http get request to a url.

Reply:
@swalladge Ah yes, since we'll want to upstream this the part about mentioning DMS explicitly could probably use some refinement :) How about this then:

>  MONGO_BACKUP_SNITCH_URL="" # Optional URL that will be used to ping a monitoring service (such as Dead Man's Snitch) upon successful completion of a backup.

Reply:
Yep that's good.